unkeyed/unkey
# fix(web): handle registered payment method

## What does this PR do?



Fixes #1367 
- Credit card payment method (original)
<img width="280" alt="credit-card-method" src="https://github.com/unkeyed/unkey/assets/51177379/ae902757-3f7d-47dd-9473-4ef38c84d205">

- Stripe Link payment method
<img width="280" alt="stripe-link-method" src="https://github.com/unkeyed/unkey/assets/51177379/2c7c1f9d-1d6c-467c-a7d3-ec83fea631b9">

- Fallback payment method
<img width="280" alt="fallback-method" src="https://github.com/unkeyed/unkey/assets/51177379/3bbf44a6-e012-435b-a281-aae9d7ca7d2b">

- No payment method  (original)
<img width="280" alt="no-method" src="https://github.com/unkeyed/unkey/assets/51177379/ece28c6a-2c96-42fb-b9a6-f0948e8a5f2f">


*If there is not an issue for this, please create one first. This is used to tracking purposes and also helps use understand why this PR exists*



## Type of change




## How should this be tested?



- Test A: Go to the Billing section. Register a payment method in your workspace and try a different method. The expected is for the UI to handle either credit-card, stripe-link or other payment methods instead of just displaying "No credit card on file.

## Checklist



### Required


### Appreciated
